packages/core/parcel-bundler/src/Asset.js

```diff
@@ -219,6 +219,19 @@ class Asset {
   }
 
   async postProcess(generated) {
+    let hasMap = false;
```

Comment:
Can we move this code to a separate method, or into the Pipeline class? If we add it here, you'll need to ensure to call `super.postProcess` from all assets that implement a `postProcess` method, as you've done for a couple already. That would be a potentially breaking change for plugins, so if we can move this outside that would be better.

Reply:
Done!